jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <j-tray-plugin
     description="2D to 1D spectral extraction."
-    :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#spectral-extraction'"
+    :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#spectral-extraction'"
```

Comment:
I thought about that, the downside is in order to have access to `vdocs`/`config` it would need to be in the init, _after_ calling super (I think), which might be a bit more subtle to remember to do for future plugins that this... but I was on the fence, so if you think that would be cleaner, I can easily switch to that instead.

Reply:
I see. Maybe this is okay for now then.

Reply:
Revisiting this - one advantage to moving the docs link to the plugin class would be that we could programmatically open a new tab from the API... `plugin.open_docs()`, or similar 🤔

Reply:
> programmatically open a new tab

Oh, dear. Astronomers have enough tabs open as it is. Do we really want to encourage that even more?